modules/hub/hub.go

```diff
@@ -136,6 +136,16 @@ func EditGithubIssue(token string, repo *repository.RepositoryStruct, number int
 	return true, nil
 }
 
+func GetGithubIssue(token string, repo *repository.RepositoryStruct, number int) (*github.Issue, error) {
```

Comment:
__[golint]__
* [golint] exported function GetGithubIssue should have comment or be unexported